lib/PostgresClient.js

```diff
@@ -1,27 +1,102 @@
 const Client = require('./Client')
+const utils = require('./utils')
 
 class PostgresClient extends Client {
+
+  charIsUpperCase(chr) {
+    return chr != chr.toLowerCase() && chr == chr.toUpperCase()
+  }
+
+  doubleQuotePascalCase(name){
+   
+    if(this.charIsUpperCase(name[0])){
```

Comment:
Isn't it a case where any use of a capital letter within a table name would need double quoting? For example `"tableName"`?